### PR TITLE
Drop "isPhpSessionsValid" requirement

### PR DESCRIPTION
**Pull request by M0rgan01**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Withdrawal of the requirement. We decided to remove it because the user will inevitably have errors before arriving on the module configuration page. The Prestashop back office requires PHP sessions, 
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | -
| Sponsor company   | -
| How to test?      | -

### CODE REVIEW

**Review by Hlavtox — APPROVED** [review]

**Review by Quetzacoalt91 — APPROVED** [review]

**Review by ga-devfront — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
